ipn/localapi/localapi.go

```diff
@@ -67,11 +83,30 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		h.serveGoroutines(w, r)
 	case "/localapi/v0/status":
 		h.serveStatus(w, r)
+	case "/localapi/v0/bugreport":
+		h.serveBugReport(w, r)
 	default:
 		io.WriteString(w, "tailscaled\n")
 	}
 }
 
+func (h *Handler) serveBugReport(w http.ResponseWriter, r *http.Request) {
+	// Require write access out of paranoia that the bugreport
```

Comment:
I think it's probably fine to restrict it to PermitRead. PermitWrite would mean people would need "sudo" for it, which seems overkill. If they can already run "tailscale status" without it (and use the network without it!), it seems fine to permit writing a log entry. There are plenty of other ways a local user could force log output.

Reply:
Make sense. Done.